platform/viewer/src/googleCloud/DicomStorePickerModal.js

```diff
@@ -33,35 +38,62 @@ class DicomStorePickerModal extends Component {
         supportsFuzzyMatching: false,
       },
     ];
-
     this.props.setServers(servers);
   };
 
   render() {
-    return (
-      <Modal
-        show={this.props.isOpen}
-        onHide={this.props.onClose}
-        aria-labelledby="ModalHeader"
-        className="modal fade themed in"
-        backdrop={false}
-        size={'md'}
-        keyboard={true}
-      >
-        <Modal.Header closeButton>
-          <Modal.Title>
-            {this.props.t('Google Cloud Healthcare API')}
-          </Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <DatasetSelector
-            setServers={this.handleEvent}
-            user={this.props.user}
-            url={this.props.url}
-          />
-        </Modal.Body>
-      </Modal>
-    );
+    if (this.props.server) {
+      return (
+        <Modal
+          show={this.props.isOpen}
+          onHide={this.props.onClose}
+          onEnter={this.setServer}
```

Comment:
We do we need the if block to add this prop?

Reply:
Agree. Removed if block with commit https://github.com/OHIF/Viewers/pull/790/commits/004e571168df85bfcb2bf635642da284fb27f812